eng/testing/linker/project.csproj.template

```diff
@@ -2,6 +2,13 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
+    <MonoAOTCompilerDir>{MonoAOTCompilerDir}</MonoAOTCompilerDir>
+    <MonoProjectRoot>{MonoProjectRoot}</MonoProjectRoot>
+    <MonoAOTCompilerTasksAssemblyPath>{MonoAOTCompilerTasksAssemblyPath}</MonoAOTCompilerTasksAssemblyPath>
+    <WasmAppBuilderTasksAssemblyPath>{WasmAppBuilderTasksAssemblyPath}</WasmAppBuilderTasksAssemblyPath>
+    <ILLinkTasksAssembly>{ILLinkTasksAssembly}</ILLinkTasksAssembly>
+    <MicrosoftNetCoreAppRuntimePackRidDir>{MicrosoftNetCoreAppRuntimePackRidDir}</MicrosoftNetCoreAppRuntimePackRidDir>
```

Comment:
This "smells" to me. Why didn't we need it before? Is it because browser-wasm apps are built differently than other projects that use the locally built runtimepack? Should we unify how this works?

Reply:
We needed something very similar before (see RuntimePackDir property bellow). Basically the property defined bellow (RuntimePackDir) will be used for building and publishing of all legs, so browser-wasm will build exactly the same as the others. This new property we are adding here, points to a subfolder inside that RuntimePackDir, and it is only used by the wasmbuildApp targets that generate the AppBundle for wasm.

Reply:
I'm not following. Why do we need both`$(RuntimePackDir)` and `$(MicrosoftNetCoreAppRuntimePackRidDir)`? Why isn't there just one property? If this points inside of `$(RuntimePackDir)`, why isn't it just based off of `$(RuntimePackDir)`?

Reply:
I have changed this now so that we only use the official property we set in our build (MicrosoftNetCoreAppRuntimePackRidDir)